app/lib/account/backend.dart

```diff
@@ -452,6 +465,73 @@ class AccountBackend {
     });
   }
 
+  /// Creates and signs a JWT token using HMAC-SHA256 signature.
```

Comment:
We have many kinds of tokens. If this is a CSRF token, we need to consistently use that name. Like `CsrfToken` or something.